scipy/stats/_continuous_distns.py

```diff
@@ -5195,6 +5195,130 @@ def _entropy(self, beta):
 halfgennorm = halfgennorm_gen(a=0, name='halfgennorm')
 
 
+class breit_wigner_gen(rv_continuous):
+    """
+    Relativistic Breit-Wigner distribution [1]_.
+ 
+    Parameters
+    ----------
+    mass : float
+        Mass of resonance        
+    width : float
+        Width of resonance 
+   
+    %(before_notes)s
+        
+    Notes
+    -----
+    We require mass > 0 and width > 0.
+
+    The width -> 0 limit is well-defined mathematically - it's proportional
+    to a Dirac function. In the m -> 0 limit, the pdf vanishes.
+    
+    The CDF was found by Mathematica:
+
+    pdf = k/((m^2 - mass^2)^2 + mass^4*alpha^2)
+    cdf = Integrate[pdf, m]
```

Comment:
This comment belongs to the comment of `_cdf`, should not be in the main docstring.